### PR TITLE
(maint) RPMs are not being properly stripped

### PR DESCRIPTION
**Pull request by underscorgan**

### CODE REVIEW

File: resources/rpm/project.spec.erb [hunk 1]
```diff
@@ -14,6 +14,16 @@
     /usr/lib64/rpm/brp-strip-static-archive %{__strip} \
     /usr/lib64/rpm/brp-strip-comment-note %{__strip} %{__objdump} \
 %{nil}
+<%- elsif !@platform.is_aix? -%>
```
Comment (mwaggett) [inline resources/rpm/project.spec.erb]: Do you need to check for cisco-wrlinux here also?

Comment (underscorgan) [reply resources/rpm/project.spec.erb]: nope, see the start of the if statement https://github.com/puppetlabs/vanagon/pull/522/files#diff-a92f582d860af1e32c4e5b9e8f9bb177R3

Comment (mwaggett) [reply resources/rpm/project.spec.erb]: ahh 👍

**Comment by mwaggett** [comment]
🙈 looks fine to me, but I don't know enough about this to give an official ✅

**Comment by sarameisburger** [comment]
Is this description accurate as to why we are doing this? https://imvoid.wordpress.com/2013/06/05/stripping-and-stopping-stripping-of-binaries-in-rpm-build/

**Comment by underscorgan** [comment]
@sarameisburger basically, we aren't currently providing debug packages, and we also want to remove debug symbols from the regular packages for size considerations. Right now rpm packages are fairly significantly larger than deb packages, with this change I was able to cut ~5MB off of the puppet-agent package for el7.

**Comment by sarameisburger** [comment]
Ah ok, that makes sense. Thanks for the detailed explanation!

**Pull request merged**